docs/site/README_theme.md

```diff
@@ -0,0 +1,86 @@
+# Easy Setup (Hugo + Netlify + Forestry)
```

Comment:
There seems to be several extraneous files in here that likely were part of the theme bundle, but don't appear to be required for operation. You may want to consider removing those.